api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -152,6 +153,12 @@ class ReservationUnitCreateSerializer(ReservationUnitSerializer, PrimaryKeySeria
         required=False,
         allow_null=True,
     )
+    pricing_terms_pk = serializers.PrimaryKeyRelatedField(
```

Comment:
Using `IntegerPrimaryKeyField` would be prefferred since this field from drf will interpret itself as string field in the api.

Reply:
I think we have the same problem with all terms fields. I could fix all of them if that is the preferred way 🤔

Reply:
`TermsOfUse` ID which is the primary key is `CharField`: https://github.com/City-of-Helsinki/tilavarauspalvelu-core/blob/main/terms_of_use/models.py#L25

I have a feeling `IntegerPrimaryKeyField` is not really the way to go here ☺️

Reply:
Oh, I stand corrected. Didn't lookup inside of the model itself. Its alright then :+1: 

(Just wonder why the id field was defined as charfield :thinking: )